sys-fs/bcachefs-tools/bcachefs-tools-1.6.1.ebuild

```diff
@@ -0,0 +1,245 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CRATES="
+	aho-corasick@1.1.2
+	anstream@0.6.11
+	anstyle-parse@0.2.3
+	anstyle-query@1.0.2
+	anstyle-wincon@3.0.2
+	anstyle@1.0.6
+	anyhow@1.0.79
+	autocfg@1.1.0
+	bindgen@0.69.4
+	bitfield@0.14.0
+	bitflags@1.3.2
+	bitflags@2.4.2
+	byteorder@1.5.0
+	cc@1.0.83
+	cexpr@0.6.0
+	cfg-if@1.0.0
+	clang-sys@1.7.0
+	clap@4.4.18
+	clap_builder@4.4.18
+	clap_complete@4.4.10
+	clap_derive@4.4.7
+	clap_lex@0.6.0
+	colorchoice@1.0.0
+	colored@2.1.0
+	either@1.9.0
+	errno-dragonfly@0.1.2
+	errno@0.2.8
+	errno@0.3.8
+	glob@0.3.1
+	heck@0.4.1
+	home@0.5.9
+	itertools@0.12.1
+	lazy_static@1.4.0
+	lazycell@1.3.0
+	libc@0.2.153
+	libloading@0.8.1
+	libudev-sys@0.1.4
+	linux-raw-sys@0.4.13
+	log@0.4.20
+	memchr@2.7.1
+	memoffset@0.8.0
+	minimal-lexical@0.2.1
+	nom@7.1.3
+	once_cell@1.19.0
+	paste@1.0.14
+	pkg-config@0.3.29
+	prettyplease@0.2.16
+	proc-macro2@1.0.78
+	quote@1.0.35
+	regex-automata@0.4.5
+	regex-syntax@0.8.2
+	regex@1.10.3
+	rpassword@7.3.1
+	rtoolbox@0.0.2
+	rustc-hash@1.1.0
+	rustix@0.38.31
+	shlex@1.3.0
+	strsim@0.10.0
+	syn@2.0.48
+	terminal_size@0.3.0
+	udev@0.7.0
+	unicode-ident@1.0.12
+	utf8parse@0.2.1
+	uuid@1.7.0
+	which@4.4.2
+	winapi-i686-pc-windows-gnu@0.4.0
+	winapi-x86_64-pc-windows-gnu@0.4.0
+	winapi@0.3.9
+	windows-sys@0.48.0
+	windows-sys@0.52.0
+	windows-targets@0.48.5
+	windows-targets@0.52.0
+	windows_aarch64_gnullvm@0.48.5
+	windows_aarch64_gnullvm@0.52.0
+	windows_aarch64_msvc@0.48.5
+	windows_aarch64_msvc@0.52.0
+	windows_i686_gnu@0.48.5
+	windows_i686_gnu@0.52.0
+	windows_i686_msvc@0.48.5
+	windows_i686_msvc@0.52.0
+	windows_x86_64_gnu@0.48.5
+	windows_x86_64_gnu@0.52.0
+	windows_x86_64_gnullvm@0.48.5
+	windows_x86_64_gnullvm@0.52.0
+	windows_x86_64_msvc@0.48.5
+	windows_x86_64_msvc@0.52.0
+"
+
+LLVM_MAX_SLOT=17
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit cargo flag-o-matic llvm multiprocessing python-any-r1 toolchain-funcs unpacker
```

Comment:
Can you port to llvm-r1?

Reply:
Will do!

Reply:
I believe I've ported all but one thing I'm not sure about, do I need `llvm_check_deps()` anymore?

Reply:
Nevermind, just double checked the docs and I see that function in llvm but not llvm-r1.

Reply:
Remember to sync live

Reply:
Done!